roles/veyepar-encoder/tasks/main.yml

```diff
@@ -3,36 +3,65 @@
   apt:
     name: "{{ item }}"
   with_items:
+  - python-psycopg2
   - libpq5
   - imagemagick
   - python3-gi
   - gstreamer1.0-tools
   - gir1.2-gstreamer-1.0
   - gir1.2-gst-plugins-base-1.0
   - gstreamer1.0-plugins-good
+  - gstreamer1.0-plugins-ugly
   - gstreamer1.0-tools
   - inkscape
+  - tmux
+  - python3-venv
+  - build-essential
+  - python3-dev
+  - libpq-dev
+
 
-# - name: git clone veyepar
-#  become: true
-#become_user:  "{{ user_name }}"
-# git:
-#     # repo: git://github.com/CarlFK/veyepar.git
-#   repo: git://github.com/CarlFK/clocky.git
-#  dest: /home/{{ user_name }}/veyepar
-# Things done to the server to this work:
-#  git clone and nfs export
-#  scp dj/local_settings.py videoteam@gw1:/srv/gw1/veyepar/dj/dj
-#  scp scripts/pw.py videoteam@gw1:/srv/gw1/veyepar/dj/scripts
-#  carl@twist:~/src/veyepar/dj/scripts/bling$ rsync -rtvP lca2017 videoteam@gw1:/srv/gw1/veyepar/dj/scripts/bling
-#  ~/src/veyepar/dj/scripts$ scp client_secrets.json videoteam@gw1:/srv/gw1/veyepar/dj/scripts
-#  apt install inkscape mplayer python3-venv
-#    build-essential python3-dev libpq-dev
 #  pip install --upgrade pip
 #
 # 2to3 -w /srv/gw1/veyepar/ve/lib/python3.5/site-packages/steve
 # 2to3 -w /srv/gw1/veyepar/ve/lib/python3.5/site-packages/vidscraper
 
+- name: git clone veyepar
+  git:
+    repo: https://github.com/CarlFK/veyepar.git
+    dest: "/home/{{ user_name }}/veyepar"
```

Comment:
I have avoided putting it anywhere but home :p

I can see that for stable packaged things that have a obvious or agreed upon proper place.

But  veyepar is a wad of code, settings, metadata, data, deployment, docs and probably a dead fish.   It is also a work in progress that gets hacked on in production.   I think putting it somewhere 'special' makes it clear that one shouldn’t expect proper separation and other such features.

OTOH, I  guess I don't really care where it lands, I shouldn't be touching these boxes anyway.